element_moseq/moseq_infer.py

```diff
@@ -197,14 +197,15 @@ class InferenceTask(dj.Manual):
     """
 
     definition = """
-    -> VideoRecording                              # `VideoRecording` key
-    -> Model                                       # `Model` key 
+    -> VideoRecording                                       # `VideoRecording` key
+    -> Model                                                # `Model` key 
     ---
-    -> PoseEstimationMethod                        # Pose estimation method used for the specified `recording_id`
-    keypointset_dir               : varchar(1000)  # Keypointset directory for the specified VideoRecording
-    inference_output_dir=''       : varchar(1000)  # Optional. Sub-directory where the results will be stored
-    inference_desc=''             : varchar(1000)  # Optional. User-defined description of the inference task
-    num_iterations=NULL           : int            # Optional. Number of iterations to use for the model inference. If null, the default number internally is 50.
+    -> PoseEstimationMethod                                 # Pose estimation method used for the specified `recording_id`
+    keypointset_dir               : varchar(1000)           # Keypointset directory for the specified VideoRecording
+    inference_output_dir=''       : varchar(1000)           # Optional. Sub-directory where the results will be stored
+    inference_desc=''             : varchar(1000)           # Optional. User-defined description of the inference task
+    num_iterations=NULL           : int                     # Optional. Number of iterations to use for the model inference. If null, the default number internally is 50.
+    task_mode='trigger'          : enum('trigger', 'load')  # Task mode for the inference task

```

Comment:
```suggestion
    task_mode='load'          : enum('load', 'trigger')  # Task mode for the inference task

```

Reply:
We typically default to `load` like [here](https://github.com/datajoint/element-calcium-imaging/blob/main/element_calcium_imaging/imaging_no_curation.py#L236)